packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -221,7 +220,7 @@ export class ModularChangeFamily
 		let valueChange: ValueChange | undefined;
 		for (const change of changes) {
 			if (change.change.valueChange !== undefined) {
-				valueChange = clone(change.change.valueChange);
+				valueChange = { ...change.change.valueChange };
```

Comment:
Why does this not go through cloneMark?

Reply:
The ValueChange type is not a Mark. It only contains two fields which don't need cloning, so doing `{ ...valueChange }` takes care of it.